### ISSUE TITLE
Regarding training data for Hindi

### ISSUE THREAD

**Issue by ashim95** [facebookresearch/LASER#64]
Hi @hoschwenk ,

We are experimenting with using LASER for translation of low-resource languages. In order to get reliable results, we have to be sure if the training data didn't contain the subsets from our test data. We are currently concerned with English-Hindi language pair. Our test data comes from WMT NewsWire 2014 which is a part of Opus collection that you have used. Is it possible for you to share the training data used for LASER for en-hi language pair? 

I have gone through the paper where its mentioned that around 288k sentences have been used for hindi. In Appendix A, there's no mention of WMT NewsWire 2014, so is it true that you have not used any data from WMT NewsWire for hindi? 
Anyways it would be great if we can have a look at the training data (for hindi) that you used so that we can create a new test set by excluding all of these sentences? 

Thanks,
Ashim

**Comment by hoschwenk** [comment]
Hello,
we have used opensubtitle2018, Global Voices and Tanzil corpus for Hindi, all from OPUS.
I don't know what includes the WMT NewsWire 2014.
I hope this helps
